tests/test_async_utils.py

```diff
@@ -44,14 +44,17 @@
     assert await flaky_coroutine() is None
 
 
+class DataHolder:
```

Comment:
would it be simpler to just use a dict or single-element list?

Reply:
yeah I thought about that but it felt a little obscure to me I guess